Demo Video URL doesnt exist anymore

### Describe the bug
The video player in the demo app is only showing a black screen.

- Library version: *1.7.1*
- Reproducible in official demo app: *yes*
- Device / Android version: *Pixel C*

### To Reproduce
Steps to reproduce the behavior, possibly in the demo app:
1. Go to the **VIDEO** tab
2. See error (video is not loading)

### Expected behavior
Video should load.

### Logs
```
E/ExoPlayerImplInternal: Source error.
    com.google.android.exoplayer2.upstream.HttpDataSource$HttpDataSourceException: Unable to connect to https://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4
        at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:281)
        at com.google.android.exoplayer2.upstream.DefaultDataSource.open(DefaultDataSource.java:250)
        at com.google.android.exoplayer2.upstream.StatsDataSource.open(StatsDataSource.java:83)
        at com.google.android.exoplayer2.source.ExtractorMediaPeriod$ExtractingLoadable.load(ExtractorMediaPeriod.java:885)
        at com.google.android.exoplayer2.upstream.Loader$LoadTask.run(Loader.java:381)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
        at java.lang.Thread.run(Thread.java:764)
     Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname clips.vorwaerts-gmbh.de not verified:
        certificate: sha1/PGVt3BMRgU1XxL3U9iQt+HAtE88=
        DN: CN=*.your-server.de
        subjectAltNames: [*.your-server.de, your-server.de]
        at com.android.okhttp.internal.io.RealConnection.connectTls(RealConnection.java:198)
        at com.android.okhttp.internal.io.RealConnection.connectSocket(RealConnection.java:149)
        at com.android.okhttp.internal.io.RealConnection.connect(RealConnection.java:112)
        at com.android.okhttp.internal.http.StreamAllocation.findConnection(StreamAllocation.java:184)
        at com.android.okhttp.internal.http.StreamAllocation.findHealthyConnection(StreamAllocation.java:126)
        at com.android.okhttp.internal.http.StreamAllocation.newStream(StreamAllocation.java:95)
        at com.android.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:281)
        at com.android.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:224)
        at com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:461)
        at com.android.okhttp.internal.huc.HttpURLConnectionImpl.connect(HttpURLConnectionImpl.java:127)
        at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.connect(DelegatingHttpsURLConnection.java:89)
        at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.connect(Unknown Source:0)
        at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.makeConnection(DefaultHttpDataSource.java:528)
        at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.makeConnection(DefaultHttpDataSource.java:436)
        at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:279)
        	... 7 more
```
Opening [https://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4](https://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4) in my browser first shows a warning, but even if I ignore it the file doesn't seem to exist anymore. So the URL should probably be replaced by something hosted directly in this repo or any other permanently working url. It would also be possible to include a small video file in the demo app.

--- a/app/src/main/java/com/otaliastudios/zoom/demo/MainActivity.java
+++ b/app/src/main/java/com/otaliastudios/zoom/demo/MainActivity.java
@@ -8,6 +8,9 @@ import android.view.View;
 import android.widget.Button;
 import android.widget.Toast;
 
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+
 import com.google.android.exoplayer2.ExoPlayerFactory;
 import com.google.android.exoplayer2.SimpleExoPlayer;
 import com.google.android.exoplayer2.source.ExtractorMediaSource;
@@ -23,9 +26,6 @@ import com.otaliastudios.zoom.ZoomLogger;
 import com.otaliastudios.zoom.ZoomSurfaceView;
 
 import org.jetbrains.annotations.NotNull;
-
-import androidx.appcompat.app.AppCompatActivity;
-import androidx.core.content.ContextCompat;
 
 
 public class MainActivity extends AppCompatActivity {
@@ -135,8 +135,9 @@ public class MainActivity extends AppCompatActivity {
         controls.show();
         DataSource.Factory dataSourceFactory = new DefaultDataSourceFactory(this,
                 Util.getUserAgent(this, "ZoomLayoutLib"));
+        Uri videoUri = Uri.parse("https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4");
         MediaSource videoSource = new ExtractorMediaSource.Factory(dataSourceFactory)
-                .createMediaSource(Uri.parse("https://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4"));
+                .createMediaSource(videoUri);
         player.prepare(videoSource);
     }
 

